### DISCUSSION TITLE
Can "SQLite Storage Back-end" in v0.7 support a DB file being updated by SQLite?

### DISCUSSION THREAD

**Discussion by foTok** [duckdb/duckdb#6380] [Q&A]
"SQLite Storage Back-end" in v0.7 support use DB files generated by SQLite, like:
```
ATTACH 'sqlite_file.db' AS sqlite (TYPE sqlite);
CREATE TABLE sqlite.tbl(i INTEGER);
INSERT INTO sqlite.tbl VALUES (1), (2), (3);
SELECT * FROM sqlite.tbl;
```
I was wondering, whether DuckDB is capable of using "sqlite_file.db" which is being updated by  a SQLite engine, or not.
If yes, DuckDB always uses the latest data, or one the data at the loaded time.

**Comment by Mytherin** [comment]
DuckDB uses SQLite under the hood to read the SQLite tables, so it has all the same guarantees/possibilities that SQLite has.
